src/cmds/pbs_ds_password.c

```diff
@@ -467,6 +467,12 @@ main(int argc, char *argv[])
 		return (-1);
 	}
 
+	if (argv[optind] != NULL) {
```

Comment:
If this functionality is not meant for general use please add a comment on why it is needed. So that even in future developers can refrain from deleting this code as it is not documented or exposed to user.

Reply:
Added